### PR TITLE
Update imperative apps list page

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://issues.redhat.com/browse/RHSTOR-5568

Screenshot 2024-02-26 at 1 51 31 PM
Screenshot 2024-02-26 at 1 51 46 PM
Screenshot 2024-02-26 at 5 36 13 PM
Screenshot 2024-02-26 at 5 37 42 PM

### CODE REVIEW

File: packages/mco/components/protected-applications/list-page.tsx [hunk 1]
```diff
@@ -137,87 +120,78 @@ const ProtectedAppsTableRow: React.FC<
...
+            className={classNames({
+              'mco-protected-applications__expanded':
+                expandableComponentType === EXPANDABLE_COMPONENT_TYPE.STATUS,
+            })}
+          />
         </Td>
         <Td translate={null} dataLabel={columnNames[4]}>
-          {getDRPolicyName(application)}
+          <Link
+            to={`/multicloud/data-services/disaster-recovery/policies?name=${drPolicyName}`}
```
[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/mco/components/protected-applications/list-page.tsx:191]: let's do it when we update DRPolicy creation page... ideally we don't redirect to "filtered" list page from a resource creation page, we either re-direct to details page or entire list...

File: packages/mco/components/protected-applications/components.tsx [hunk 2]
```diff
@@ -81,19 +100,19 @@ export const EmptyRowMessage: React.FC = () => {
...
-        <Trans t={t}>
-          You do not have any protected applications yet, to add disaster
-          recovery protection to your applications start by clicking on the{' '}
-          <strong>Enroll application</strong> button.
-        </Trans>
-      </EmptyStateBody>
-    </EmptyState>
+    <EmptyPage
+      title={t('No protected applications')}
+      t={t}
```
[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/mco/components/protected-applications/components.tsx]: done...

**Review by GowthamShanmugam — APPROVED** [review]

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**